intelij-plugin/src/main/gen/tech/pantheon/yanginator/plugin/psi/YangAbsolutePath.java

```diff
@@ -17,10 +17,10 @@
 
 public interface YangAbsolutePath extends YangNamedElement {
 
-    @NotNull
```

Comment:
wrong formatting, it should have 4 spaces like before, please fix formatting in all affected generated files.